desktop-app/app/components/UserPreferences/index.js

```diff
@@ -106,6 +107,15 @@ export default function UserPreference({
           />
         </div>
       </div>
+      <div className={cx(commonStyles.sidebarContentSectionContainer)}>
+        <ScreenShotSavePreference
+          screenShotSavePath={
+            userPreferences.screenShotSavePath ||
+            `Desktop/Responsively-Screenshots`
```

Comment:
Can you please change this to the full path? So that users can just copy-paste the path if required.

Reply:
Thanks @manojVivek , full path means have to get the whole path from the ipc renderer via event ? below i have attached the screenshot for user preferences section

![screenshot](https://user-images.githubusercontent.com/38885946/87248241-4f2a1b80-c40d-11ea-977c-48f734b8a936.PNG)

Reply:
I think you can move the below logic to a util function and use it both here and in `app/menu.js` file too

```
const getDefaultScreenshotpath = () => path.join(
                os.homedir(),
                `Desktop/Responsively-Screenshots`,
                directoryPath
              );
```

Reply:
Thanks for the feedback, i've moved the logic into userPreferenceSettings class.